Added the QALD-10 dataset

LongquanJiang opened the pull request.

RicardoUsbeck commented:
1) Could you call that dataset "QALD-10 Test" please? 
2) Shouldn't we also add the train split?

RicardoUsbeck commented:
see questions above

LongquanJiang commented:
> 

qald10_train = load_dataset("qald/qald-10.py", "qald10", split="train")
qald10_test = load_dataset("qald/qald-10.py", "qald10", split="test")

qald10_train is for loading qald-9-plus dataset as training set
qald10_test is for loading qald-10 as test set

Is it ok?

RicardoUsbeck commented on qald/qald-10.py:
```diff
@@ -92,4 +102,8 @@ def _generate_examples(self, data_file, **kwargs):
                 question["answers"] = json.dumps(question["answers"])
 
+                if kwargs["split"]== "test":
+                    print("asda")
```
remove

RicardoUsbeck reviewed (commented):
I think the naming should be QALD-10 Train and QALD-10 Test
